docs/HowToAddnewBoard.md

```diff
@@ -211,3 +211,15 @@ This is moved to .config at executing configure.sh and used to build TizenRT.
 This is a build options set like setting of including path, ARM build options, linker script name and so on.  
 There is the DOWNLOAD definition to support "make download xx" command too.
 
+### Linker Variables
+
+Linker script has memory configuration information.  
+file path : *build/configs/<BOARD_NAME>/scripts/<SCRIPTS_NAME>.ld* 
+
+In addition to the SECTION information, add the following variables as well to mark the following:  
+1. `_sbss`: Start of .bss section
+2. `_ebss`: End+1 of .bss section
+3. `_sidle_stack`: Start of idle stack
+4. `__sint_heap_start`: Start of heap in internal RAM region
+5. `__sext_heap_start`: Start of heap in external RAM region
```

Comment:
They need some guidance about where chipset vendor should locate them.
Let's add two examples for artik and rtl8720e.